packages/ui/src/layout/MainLayout/Header/ProfileSection/index.jsx

```diff
@@ -258,24 +259,24 @@ const ProfileSection = ({ username, handleLogout }) => {
                                                 <ListItemButton
                                                     sx={{ borderRadius: `${customization.borderRadius}px` }}
                                                     onClick={() => {
-                                                        getAllChatflowsApi.request()
+                                                        exportAllApi.request()
```

Comment:
@HenryHengZJ 

Need to modify this to call out the dialogbox input you wanted instead of straight away making request.